tests/km_core_tests.bats

```diff
@@ -1103,7 +1103,16 @@ fi
    tries=5
    while [ ! -S ${MGMTPIPE} ] && [ $tries -gt 0 ]; do sleep 1; tries=`expr $tries - 1`; done
    assert [ $tries -gt 0 ]
-   pid=`pidof hello_html_test$ext`
+   pidlist=`pidof hello_html_test$ext`
+   # Find our pid if pidof returned multiple pids
+   pid=""
+   for p in $pidlist; do
+      if lsof -p $p | grep " 729u .* $MGMTPIPE " >/dev/null 2>&1; then
```

Comment:
`|` -> `|&`, `grep -q` would make redirects unnecessary

Reply:
I'll change that.  It occurs to me that lsof is probably not in the test container.
I will switch over to looking at /proc/$pid/fd/729
I wonder if the readlink command is available.

Reply:
what is 729u. 

MGTPIPE is /tmp/mgmtpipe.$$. $$ is the shell pid. Is this ok to create uniqueness?

Reply:
`lsof` is not there, you are right. We can put it there but it will require to rebuild buildenv image. 

`readlink` is there, it is part of coreutils, so it's there on most systems, even very small ones.

Reply:
> what is 729u.
> 
> MGTPIPE is /tmp/mgmtpipe.$$. $$ is the shell pid. Is this ok to create uniqueness?

The km management pipe is always on fd 729.